definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1272,5 +1272,12 @@
     "CaseEventID": "FR_GeneralApplicationDirections",
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseEventID": "FR_updateRespondentContactDetails",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
   }
```

Comment:
We have a failing test as events need R permission atleast for each of the other roles as well
(solicitor, judge, caseworker). So can we add in:

{
    "LiveFrom": "01/09/2020",
    "CaseTypeID": "FinancialRemedyContested",
    "CaseEventID": "FR_updateRespondentContactDetails",
    "UserRole": "caseworker-divorce-financialremedy-courtadmin",
    "CRUD": "R"
  },
  {
    "LiveFrom": "01/09/2020",
    "CaseTypeID": "FinancialRemedyContested",
    "CaseEventID": "FR_updateRespondentContactDetails",
    "UserRole": "caseworker-divorce-financialremedy-solicitor",
    "CRUD": "R"
  },
  {
    "LiveFrom": "01/09/2020",
    "CaseTypeID": "FinancialRemedyContested",
    "CaseEventID": "FR_updateRespondentContactDetails",
    "UserRole": "caseworker-divorce-financialremedy-judiciary",
    "CRUD": "R"
  },